storage/src/main/java/com/tencent/rss/storage/common/Storage.java

```diff
@@ -42,6 +42,8 @@
 
   ShuffleWriteHandler getOrCreateWriteHandler(CreateShuffleWriteHandlerRequest request) throws IOException;
 
+  CreateShuffleWriteHandlerRequest getCreateWriterHandlerRequest(String appId, int shuffleId, int partition);
```

Comment:
why this api is necessary?

Reply:
When we write hdfs failed, we need use the request to create local handler.

Reply:
ok